### ISSUE TITLE
0.46 Breaks app with firebase/angularfire (0.45 works fine)

### ISSUE THREAD

**Issue by gigocabrera** [driftyco/ionic-app-scripts#455]
#### Short description of the problem:

My firebase/angularfire app builds and works fine with app-scripts 45. After I installed 46 I got the following error: `Cannot find namespace 'firebase'` and the app won't build/work.

After rolling back to 45 everything works as expected again.

#### What behavior are you expecting?

I expect to have app-scripts 46 to work with Firebase just as 45 did. 

**Steps to reproduce:**
1. Clone repository https://github.com/gigocabrera/MoneyLeash2
2. Install app-scripts 45 and `ionic serve`
3. Now install app-scripts 46 and `ionic serve`

**Which @ionic/app-scripts version are you using?**
I'm using 45 because 46 breaks my app

**Comment by Chris1308** [comment]
Try to set firebase to Version 3.3.0. That fixed the Problem for me (after deleting node_modules, www, tmp and then a fresh npm install).

**Comment by tja4472** [comment]
Hi

Try updating angularfire to beta.6
https://github.com/angular/angularfire2/blob/master/CHANGELOG.md

```
imports: add firebase imports to all places that reference firebase namespace (c3a954c), closes #525
```

**Comment by gigocabrera** [comment]
Setting firebase to 3.3.0, angularfire to beta.6 and app-scripts to 46 did the trick. Thank you very much @Chris1308 and @tja4472
